src/main.py

```diff
@@ -0,0 +1,51 @@
+import json
+import os
+import random
+
+from rich import box
+from rich.table import Column, Table
+
+from src.data.data import load_data_std, load_data_img
+from src.models.t5_trainer import T5Trainer
+from src.parser_args import parse_args
```

Comment:
Rename the file args_parser.py or parse_args.py

Reply:
ok